internal/provider/data_source_http_test.go

```diff
@@ -145,26 +146,31 @@ func TestDataSource_utf16_200(t *testing.T) {
 	})
 }
 
-func TestDataSource_x509cert(t *testing.T) {
-	testHttpMock := setUpMockHttpServer()
-	defer testHttpMock.server.Close()
-
-	resource.UnitTest(t, resource.TestCase{
-		ProtoV6ProviderFactories: protoV6ProviderFactories(),
-		Steps: []resource.TestStep{
-			{
-				Config: fmt.Sprintf(`
-							data "http" "http_test" {
-  								url = "%s/x509-ca-cert/200"
-							}`, testHttpMock.server.URL),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("data.http.http_test", "response_body", "pem"),
-					resource.TestCheckResourceAttr("data.http.http_test", "status_code", "200"),
-				),
-			},
-		},
-	})
-}
+// TODO: This test fails under Terraform 0.14. It should be uncommented when we
+// are able to include Terraform version logic within acceptance tests, or when
```

Comment:
> are able to include Terraform version logic

https://github.com/hashicorp/terraform-plugin-sdk/issues/776 is an upstream testing framework issue reference if that helps

Reply:
Thanks. Have added reference.